src/usr/local/www/system_usermanager_settings.php

```diff
@@ -177,15 +174,19 @@
 
 $section = new Form_Section('Settings');
 
+list($d_hr, $d_min) = [int($g['default_session_timeout_mins']/60), $g['default_session_timeout_mins'] % 60];
+if ($d_min == 0) {
+	$d = sprintf(gettext("%d hours (%d minutes)"), $d_hr, $g['default_session_timeout_mins']);
+} else {
+	$d = sprintf(gettext("%d hours %d mins (%d minutes)"), $d_hr, $d_min, $g['default_session_timeout_mins']);
+}
 $section->addInput(new Form_Input(
 	'session_timeout',
 	'Session timeout',
 	'number',
-	$pconfig['session_timeout'],
-	['min' => 0]
-))->setHelp('Time in minutes to expire idle management sessions. The default is 4 '.
-	'hours (240 minutes). Enter 0 to never expire sessions. NOTE: This is a security '.
-	'risk!');
+	$pconfig['session_timeout']
+))->setHelp(sprintf(gettext('Time in minutes to expire idle management sessions. Leave empty for the default timeout ' .
+	'of %s. Enter 0 to never expire sessions. NOTE: Non-expiry is a security risk!'), $d));
```

Comment:
The note text needs changed. The last two sentences could be better, maybe:
"A value of 0 will never expire idle sessions, which is a security risk"